### PR TITLE
ath79: Add support for TP-Link TL-WR941N/ND v2/v3

### PR DESCRIPTION
**Pull request by 981213**
Specification:
- SoC: Atheros AR9132
- Flash: 4 MB
- RAM: 32 MB
- Ethernet: Marvell 88E6060 with 5 FE ports.

Flash instruction:
  Upload the generated factory firmware on web interface.

Signed-off-by: Chuanhong Guo <gch981213@gmail.com>

### CODE REVIEW

File: target/linux/ath79/dts/ar9132_tplink_tl-wr941-v2.dts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+			};
+
+			partition@20000 {
+				label = "firmware";
+				reg = <0x020000 0x3d0000>;
+			};
+
+			art: partition@3f0000 {
+				label = "art";
+				reg = <0x7f0000 0x010000>;
```
Comment (hanipouspilot) [inline target/linux/ath79/dts/ar9132_tplink_tl-wr941-v2.dts]: reg = <0x3f0000 0x010000>;

Comment (981213) [reply target/linux/ath79/dts/ar9132_tplink_tl-wr941-v2.dts]: Aw...
Fixed.

**Comment by pkgadd** [comment]
Tested successfully on a TL-WR941ND v2.4.

**Comment by mkresin** [comment]
Was this board ever supported in ar71xx?

**Comment by pepe2k** [comment]
@mkresin 
https://wiki.openwrt.org/toh/hwdata/tp-link/tp-link_tl-wr941nd_v2

**Comment by mkresin** [comment]
merged, thanks!

**Pull request closed without merging**